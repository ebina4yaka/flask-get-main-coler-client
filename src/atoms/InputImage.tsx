import { ChangeEvent, ReactElement } from 'react'

type Props = {
  handleChange: (event: ChangeEvent<HTMLInputElement>) => void
}

export default function InputImage(props: Props): ReactElement {
  const { handleChange } = props
  return (
    <div>
      <input type="file" accept="image/*" onChange={handleChange} />
    </div>
  )
}
