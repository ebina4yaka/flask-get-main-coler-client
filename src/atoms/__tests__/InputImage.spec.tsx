import renderer from 'react-test-renderer'
import InputImage from '../InputImage'

test('Snapshot test for InputImage', () => {
  const tree = renderer.create(<InputImage handleChange={() => {}} />).toJSON()
  expect(tree).toMatchSnapshot()
})
