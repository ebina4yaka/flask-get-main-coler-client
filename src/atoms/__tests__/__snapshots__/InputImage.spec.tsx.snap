// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for InputImage 1`] = `<div />`;
